NateGoSearch/NateGoSearchQuery.php

```diff
@@ -234,10 +234,14 @@ public function query($keywords)
 		$misspellings = $this->getPopularReplacements($keywords, $misspellings);
 
 		$keywords = $this->normalizeKeywordsForSearching($keywords);
-		$keywords_hash = sha1($keywords);
+		$unique_hash = sha1($keywords.serialize($this->document_types));
```

Comment:
I think we shouldn't. The order of the keywords might play a role in proximity ranking. It's also how it's always been and I want to avoid introducing "improvements" rather than just fixing this one bug.